python/hail/expr/functions.py

```diff
@@ -3494,3 +3494,60 @@ def get_sequence(reference_genome, contig, position, before=0, after=0) -> Strin
         raise TypeError("Reference genome '{}' does not have a sequence loaded. Use 'add_sequence' to load the sequence from a FASTA file.".format(reference_genome.name))
     return _func("getReferenceSequence({})".format(reference_genome.name), tstr,
                  contig, position, before, after)
+
+@typecheck(reference_genome=reference_genome_type,
+           contig=expr_str)
+def is_valid_contig(reference_genome, contig) -> BooleanExpression:
+    """Returns ``True`` if `contig` is a valid contig name in `reference_genome`.
+
+    Examples
+    --------
+
+    >>> hl.eval_expr(hl.is_valid_contig('GRCh37', '1'))
+    True
+
+    >>> hl.eval_expr(hl.is_valid_contig('GRCh37', 'chr1'))
+    False
+
+    Parameters
+    ----------
+    reference_genome : :obj:`str` or :class:`.ReferenceGenome`
+        Reference genome to use.
+    contig : :class:`.Expression` of type :py:data:`.tstr`
+        Contig name to test.
+
+    Returns
+    -------
+    :class:`.Expression` of type :py:data:`.tbool`
+    """
+    return _func("isValidContig({})".format(reference_genome.name), tbool, contig)
+
+@typecheck(reference_genome=reference_genome_type,
+           contig=expr_str,
+           position=expr_int32)
+def is_valid_locus(reference_genome, contig, position) -> BooleanExpression:
```

Comment:
what do you think about flipping this so that reference_genome is at the end and is `'default'` by default? Seems like most of the other things that use a reference genome do that. Up to you, though.